app/app/src/main/java/com/morning5/vocabularytrainer/OverviewActivity.java

```diff
@@ -44,11 +70,11 @@ private void showVocabularies() {
             return;
         }
 
-        ArrayList<VocabularyData> list = new ArrayList<VocabularyData>();
+        list = new ArrayList<VocabularyData>();
         OverviewAdapter overviewAdapter = new OverviewAdapter(list, this);
 
         while (cursor.moveToNext()) {
-            VocabularyData vocabularyData = new VocabularyData(cursor.getString(1), cursor.getString(2),cursor.getString(3),cursor.getString(4));
+            VocabularyData vocabularyData = new VocabularyData(cursor.getString(0), cursor.getString(1), cursor.getString(2),cursor.getString(3),cursor.getString(4));
```

Comment:
Either set the column name in the select --> `SELECT _id, word1, language1, word2, language2 FROM Word`;
or use `cursor.getString(cursor.getColumnIndex(WordContract.Word._ID), ....`